HumanControls/JoystickAxis.cpp

```diff
@@ -27,23 +30,34 @@ double JoystickAxis::calcResult(int rawValue)
 {
     double value = rawValue;
 
-    if (value >= m_min && value >= (m_range + m_min))
+    //General calculations
+    double deadMin = m_center - m_halfDeadZoneSize;
+    double deadMax = m_center - m_halfDeadZoneSize;
+
+    //Assumes that this is the raw value the joystick returns when it is untouched
```

Comment:
I would figure the joystick would be at "center" when untouched. Not sure about this.